keps/sig-auth/20190607-certificates-api.md

```diff
@@ -235,6 +236,20 @@ Cluster admins can either:
 2. grant signer-specific approval permissions using the bootstrap roles starting in 1.18
 3. disable the approval-authorizing admission plugin in 1.18 (if they don't care about partitioning approver rights)
  
+#### Supporting intermediate certificates.
+It's important that the Certificates API fully supports using an intermediate certificate as the signer, 
+including the case where the signer is itself signed by an intermediate. As a result, the user must be able 
+to specify that the signing certificates are provided to the client, in which case their PEM values would 
+be appended to the Certificate in the CertificateSigningRequestStatus. Conforming clients will parse the 
```

Comment:
I recommend unambiguously specify the order in which intermediate certs are added into CertificateSigningRequest's `status` field *or* stating that the order is arbitrary / random.